### PR TITLE
Add Dependency Size

### PR DESCRIPTION
**Pull request by SaraVieira**
**What kind of change does this PR introduce?**
Adds all dependency sizes to a dependency

**Checklist**:
- [ ] Documentation
- [ ] Tests
- [x] Ready to be merged

screenshot 2018-10-16 at 18 49 06
screenshot 2018-10-16 at 18 49 09

### CODE REVIEW

**Comment by SaraVieira** [comment]
closes #1023

**Comment by CompuIves** [comment]
Oh man this is awesome! I love this!

I'm thinking that we should maybe color code it or hide it by default, because it may be a bit messy for people to read right now (people mainly use it to see the version and now they have to scan more to get that information). Maybe we do something in the direction of a popover or maybe a way to expand the dependency to show advanced details? Like if you click on it it becomes bigger and shows you the tag (if applicable), the size and the name.

**Comment by SaraVieira** [comment]
Updated with the following styles:
screenshot 2018-10-17 at 15 45 04
screenshot 2018-10-17 at 15 44 58

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/VersionEntry/index.js [hunk 1]
```diff
@@ -44,35 +80,55 @@ export default class VersionEntry extends React.PureComponent {
...
+              <Icon onClick={this.handleRefresh}>
+                <RefreshIcon />
+              </Icon>
+              <Icon onClick={this.handleRemove}>
+                <CrossIcon />
+              </Icon>
+            </IconArea>
+          )}
+        </EntryContainer>
+        {open ? (
```
Comment (franklinjavier) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/VersionEntry/index.js]: To improve readability and maintain the pattern (like `hovering && (` above)

```diff
- {open ? () : null}
+ {open && ()}
```

**Pull request merged**